sqlglot/dialects/bigquery.py

```diff
@@ -727,6 +727,18 @@ class Generator(generator.Generator):
             "within",
         }
 
+        def struct_sql(self, expression: exp.Struct) -> str:
+            args = []
+            for expr in expression.expressions:
+                if isinstance(expr, self.KEY_VALUE_DEFINITONS):
```

Comment:
key_value_definitions, typo in the original

Reply:
good catch